plugins/extractors/tableau/tableau_test.go

```diff
@@ -36,6 +36,17 @@ func TestInit(t *testing.T) {
 
 		assert.Equal(t, plugins.InvalidConfigError{}, err)
 	})
+	t.Run("should return no error for valid config", func(t *testing.T) {
```

Comment:
can you add an extra test similar to this 🙏 
https://github.com/odpf/meteor/blob/main/plugins/extractors/metabase/metabase_test.go#L38-L64

Reply:
hey, I already added a test for the validation right, and in client I have made changes to logic where if auth_token is present, then it skips generating one. Can you let me know what scope should I test it for??

Reply:
@StewartJingga added few more tests with various combinations.